app/views/hub/users/edit.html.erb

```diff
@@ -16,15 +16,22 @@
         <% end %>
       <% end %>
 
-      <%= form_with model: [:hub, @user], method: :put, local: true, builder: VitaMinFormBuilder, html: { class: 'form-card form-card--long' } do |f| %>
```

Comment:
form-card--long changes the size of the labels on the inputs (this class provides an override to make labels _smaller_ than the default enormous ones on customer-facing honeycrisp, so it makes them _bigger_ than our default admin styles when applied here.